docs/usage.md

```diff
@@ -67,7 +71,7 @@ And Django 1.x `urls.py`:
 from flags.urls import flagged_url
 
 urlpatterns = [
-    flagged_url('MY_FLAG', r'^a-url$', view_requiring_flag, state=True),
+    flagged_url('FLAG_WITH_ANY_CONDITIONS', r'^a-url$', view_requiring_flag, state=True),
 ]
 ```
 
```

Comment:
*Checking* flags is the same regardless of their conditions. I think I'd rather make that clear, since all the examples are around checking, rather than defining.